packages/static/src/progress-bar/styles/progress-bar.module.css

```diff
@@ -0,0 +1,109 @@
+@import '../../theme/theme.module.css';
+
+@keyframes progressLoading {
+  from {
+    background-position-x: 200%;
+  }
+  to {
+    background-position-x: 0;
+  }
+}
+
+@keyframes progressSlideLoading {
+  0% {
+    left: 0%;
+    right: 100%;
+    width: 0%;
+  }
+  10% {
+    left: 0%;
+    right: 75%;
+    width: 25%;
+  }
+  90% {
+    right: 0%;
+    left: 75%;
+    width: 25%;
+  }
+  100% {
+    left: 100%;
+    right: 0%;
+    width: 0%;
+  }
+}
+
+.progressBarWrapper {
+  width: 100%;
+}
+
+.progressBarLabel {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: var(--spacing-2);
+}
+
+.progressBarLabelSpan {
+  font-weight: var(--font-weight-normal);
+  font-size: var(--font-size-label1);
```

Comment:
wrong variable name. no theme variable is named like this

Reply:
fixed